generator_ripe_caida.sh

```diff
@@ -8,7 +8,7 @@ set -exuo pipefail
 # Use cpu-efficient tools across 2-4 cores
 
 tstamp=$(date +%Y%m%d%H%M%S)
-caida_tstamp=20201001
+caida_tstamp=$(date +%Y%m%01)
```

Comment:
This would break because they don't update monthly. We need a smarter logic here.

Reply:
See http://data.caida.org/datasets/as-organizations/

Reply:
Good call. I will take another stab at this.

Reply:
Even if we were to update monthly, we need to manage race conditions around the time of publishing from Caida

Reply:
How frequently is the github action run?

Reply:
Weekly: https://github.com/ooni/asn-db-generator/blob/master/.github/workflows/generate.yml#L5

Reply:
(After a few weeks of inactivity, GH disables automatic runs)